tests/e2e/docs/vmc_setup.md

```diff
@@ -0,0 +1,108 @@
+# E2E Tests on VMC environment
+
+## Configuring VCenter
+
+### 1. Create namespace
+
+#### Go to Workload Management
+
+![Go to Workload Management](images/create_namespace_step_1.png)
+
+#### Click on Create Namespace and enter necessary inputs
+
+![Create Namespace](images/create_namespace_step_2.png)
+
+### 2. Create a VM Storage Policy and assign it to the namespace created above
+
+![Create Storage Policy](images/create_vmc_storage_policy.png)
+
+#### Assign storage policy to the namespace
+
+![Assign storage policy to namespace](images/assign_policy_to_ns.png)
+
+## Running e2e Tests from pop machine
+
+The section outlines how to set the env variable for running e2e test.
+
+### Building e2eTest.conf
+
+    [Global]
+    insecure-flag = "true"
+    hostname = "<VC_IP>"
+    user = "<USER>"
+    password = "<PASSWORD>"
+    port = "443"
+    vmc-devopsuser = "<DEVOPS_USER>"
+    vmc-devopspassword = "<PASSWORD>"
+    vmc-cloudadminuser = "<CLOUD_USER>"
+    cloudadminpassword = "<PASSWORD>"
+    datacenters = "<Datacenter_Name>"
```

Comment:
this is required or optional

Reply:
not sure if any regression tests are using this value, to avoid regression issues I have captured it.

Reply:
mention it as optional then

Reply:
done